endpoints.json

```diff
@@ -30,5 +30,119 @@
         }
       ]
     }
+  },
+  "GET /api/reviews/:review_id": {
+    "description": "Serves a review object of specified review_id including number of associated comments",
+    "queries": [],
+    "exampleResponse": {
+      "review": {
+        "review_id": 1,
+        "title": "Agricola",
+        "category": "euro game",
+        "designer": "Uwe Rosenburg",
+        "owner": "mallionaire",
+        "review_body": "Farmyard fun!",
+        "review_img_url": "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png",
+        "created_at": "2021-01-18T10:00:20.514Z",
+        "votes": 1,
+        "comment_count": 0
+      }
+    }
+  },
+  "PATCH /api/reviews/:review_id": {
+    "description": "Increments/decrements vote count of specified review entry by specified value and serves updated review object",
+    "queries": [],
```

Comment:
add one property for patch or post -
exampleRequest: { inc_votes:1 }
This is for users to know what kind of body they need to send in order to make the request successfully